datafusion/sql/src/planner.rs

```diff
@@ -3170,10 +3170,10 @@ mod tests {
                      ) AS a
                    ) AS b";
         let expected = "Projection: b.fn2, b.last_name\
-                        \n  Projection: b.fn2, b.last_name, b.birth_date, alias=b\
-                        \n    Projection: a.fn1 AS fn2, a.last_name, a.birth_date, alias=b\
```

Comment:
yeah looking at these old plans now this selecting a just to alias them as `b` is a little strange